source/manifest.json

```
@@ -8,7 +8,7 @@
 	"applications": {
 		"gecko": {
 			"id": "{a4c4eda4-fb84-4a84-b4a1-f7c1cbf2a1ad}",
-			"strict_min_version": "102.0"
+			"strict_min_version": "119.0"
```

This should be 121, no?
```suggestion
			"strict_min_version": "121.0"
```